docs/inference_protocols.md

```diff
@@ -70,7 +77,7 @@ Triton allows the on-wire compression of request/response messages by exposing f
 
 * `--grpc-infer-response-compression-level`
 
-For client-side documentation, see [Client-Side GRPC Compression](https://github.com/triton-inference-server/client/tree/main#compression)
+For client-side documentation, see [Client-Side GRPC Compression](https://github.com/triton-inference-server/client/tree/main#compression-1)
```

Comment:
Why are you adding the -1 in the end?

Reply:
The compression without -1 links to http compression and compression-1 links to the second compression sub-heading for grpc.
https://github.com/triton-inference-server/client/tree/main#compression
https://github.com/triton-inference-server/client/tree/main#compression-1

I too was not aware of this way of indexing.

Reply:
TIL